packages/cloud/src/cloud/withCloudOverrides.ts

```diff
@@ -0,0 +1,40 @@
+import { createClient } from '@portive/client';
+import { Value, WithPlatePlugin } from '@udecode/plate-core';
+import { createUploadStore } from '../upload/createUploadStore';
+import { finishUploads } from './finishUploads';
+import { getSaveValue } from './getSaveValue';
+import { CloudEditor, CloudPlugin, FinishUploadsOptions } from './types';
+import { uploadFiles } from './uploadFiles';
+
+export function withCloudOverrides<
+  V extends Value = Value,
+  E extends CloudEditor<V> = CloudEditor<V>
```

Comment:
```suggestion
  E extends PlateEditor<V> = PlateEditor<V>
```
otherwise I think it will throw a ts error in the `plugins` array

Reply:
Working example:

```ts
export const withPlateYjs = <
  V extends Value = Value,
  E extends PlateEditor<V> = PlateEditor<V>,
  EE extends E & PlateYjsEditor<V> = E & PlateYjsEditor<V>
>(
  e: E,
) => {
  const editor = e as EE
```

Reply:
This is stuck because \`Conversion of type 'E' to type 'EE' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.  
'EE' could be instantiated with an arbitrary type which could be unrelated to 'E'.ts(2\` which is related to the ReactEditor/TReactEditor in the conversation above.

I think when we find a fix for that, this will work too. I'll leave it for now.